File: GTTS/INL.PersonService.Models/IGetPersonsTraining_Item.cs
using System;

namespace INL.PersonService.Models
{
    public interface IGetPersonsTraining_Item 
    {
        long TrainingEventID { get; set; }
        string Name { get; set; }
        long PersonID { get; set; }
        string ParticipantType { get; set; }
        DateTime? EventStartDate { get; set; }
        DateTime? EventEndDate { get; set; }
        string BusinessUnitAcronym { get; set; }
        string TrainingEventRosterDistinction { get; set; }
        bool? Certificate { get; set; }
        string TrainingEventStatus { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/iget-persons-training_item.ts




export interface IGetPersonsTraining_Item {
  
	TrainingEventID: number;
	Name: string;
	PersonID: number;
	ParticipantType: string;
	EventStartDate?: Date;
	EventEndDate?: Date;
	BusinessUnitAcronym: string;
	TrainingEventRosterDistinction: string;
	Certificate?: boolean;
	TrainingEventStatus: string;

}

